group/edwards25519/scalar.go

```diff
@@ -24,7 +24,12 @@ import (
 // The scalars are GF(2^252 + 27742317777372353535851937790883648493).
 
 type scalar struct {
-	v [32]byte
+	v     [32]byte
```

Comment:
This increases the size of a scalar from 32 bytes to 40 bytes (32 + one pointer). Why? It seems like for ed25519, Group() could return a pointer to a fixed kyber.Group which is shared by all instances.

(Same for point.)

Reply:
No because now the suite has a `random.Stream` that can produces deterministic output when using `XOF` or `Hash()`. It's again related to that `Suite` functionality... A lot of things would be simpler if we killed that thing ;)